app/src/main/java/org/schabi/newpipe/database/BasicDAO.java

```diff
@@ -14,13 +13,10 @@
 @Dao
 public interface BasicDAO<Entity> {
     /* Inserts */
-    @Insert(onConflict = OnConflictStrategy.ABORT)
```

Comment:
Why this change? Could you document such changes when you open a PR?

Reply:
Sorry, my bad.